.github/workflows/build.yml

```diff
@@ -27,7 +27,6 @@ jobs:
       - name: Print lines of code
         run: cloc --vcs git --exclude-dir Resources,store,test,Properties --include-lang C#,XAML
 
-
```

Comment:
These double newlines are intentional to ease maintenance burdens on large workflow files. We have a [custom linter](https://github.com/bitwarden/gh-actions/tree/master/lint-workflow) that catches most of our standards and the rest are documented in Confluence (which should probably be moved to the new public contributing docs)